datalad/core/distributed/clone.py

```diff
@@ -581,21 +581,141 @@ def postclonecfg_ria(ds, props):
+            from datalad.cmd import (
+                GitRunner,
+                WitlessRunner,
+                StdOutCapture
+            )
+
+            with make_tempfile(content=config_content) as cfg_file:
+                runner = WitlessRunner(env=GitRunner.get_git_environ_adjusted())
+                try:
+                    result = runner.run(
+                        ['git', 'config', '-f', cfg_file,
+                         'datalad.ora-remote.uuid'],
+                        protocol=StdOutCapture
+                    )
+                    org_uuid = result['stdout'].strip()
+                except CommandError:
+                    # doesn't contain what we are looking for
+                    lgr.debug("Found no UUID for ORA special remote at "
+                              "'origin'")
+
+        # Now, enable it. If annex-init didn't fail to enable it as stored, we
+        # wouldn't end up here, so enable with store URL as suggested by the URL
+        # we cloned from.
+        if org_uuid:
+            srs = ds.repo.get_special_remotes()
+            if org_uuid in srs.keys():
+                # TODO: - Double-check autoenable value and only do this when
+                #         true?
+                #       - What if still fails? -> Annex shouldn't change config
+                #         in that case
+
+                # we only need the store:
+                new_url = props['source'].split('#')[0]
+                try:
+                    ds.repo.enable_remote(srs[org_uuid]['name'],
+                                          options=['url={}'.format(new_url)]
+                                          )
+                    lgr.info("Reconfigured %s for %s",
+                             srs[org_uuid]['name'], new_url)
+                    # update ora_remotes for considering publication dependency
+                    # below
+                    ora_remotes = [s for s in
+                                   ds.siblings('query',
+                                               result_renderer='disabled')
+                                   if s.get('annex-externaltype', None) ==
+                                   'ora']
+                except CommandError as e:
+                    lgr.debug("Failed to reconfigure ORA special remote: %s",
+                              e)
+            else:
+                lgr.debug("Unknown ORA special remote uuid at 'origin': %s",
+                          org_uuid)
+    if ora_remotes:
+        if len(ora_remotes) == 1:
+            yield from ds.siblings('configure',
+                                   name='origin',
+                                   publish_depends=ora_remotes[0]['name'],
+                                   result_filter=None,
+                                   result_renderer='disabled')
+        else:
+            lgr.warning("Found multiple ORA remotes. Couldn't decide which "
+                        "publishing to origin should depend on: %s",
```

Comment:
and what should the user do when is warning displayed - should it may be suggest something ? (remove one?)